### DISCUSSION TITLE
Found the wifi password, when i attempt to connect to the wifi by creating a new network, it says error configuing network

### DISCUSSION THREAD

**Discussion by thebossjeff** [3kh0/ext-remover#457] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so, it will not allow me to create a new connection. I'm sure I am just making a simple mistake, someone please tell me what I am doing wrong

**Comment by 3kh0** [comment, marked as answer]
Just seems like the password is wrong, so nothing much.
